.travis.yml

```diff
@@ -4,31 +4,16 @@ node_js:
   - "0.10"
   - "4"
   - "5"
-env:
-  - FIREFOX_VERSION=nightly
-#  - FIREFOX_VERSION=aurora
-#  - FIREFOX_VERSION=beta
-#  - FIREFOX_VERSION=release
-
+addons:
+  firefox: latest-beta
 notifications:
   irc: "irc.mozilla.org#jpm"
-
 before_install:
-  - "export DISPLAY=:99.0"
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
   - npm install -g npm
-
 install:
   - npm install
-
-before_script:
-  - pwd
-  - cd ..
-  - wget "https://download.mozilla.org/?product=firefox-nightly-latest&os=linux64&lang=en-US" -O firefox.tar.bz2 && tar xvf firefox.tar.bz2
```

Comment:
This is an odd request but can you leave this command commented out prefixed with something like 'this is how you get nightly' ? Unfortunately, it is not documented very well and I'm worried we'll forget how to do it if we need to switch back to nightly.